app/Http/Requests/WebhookUpdateRequest.php

```diff
@@ -29,18 +29,20 @@ public function rules()
             'bot_id' => 'required',
             'room_name' => 'required',
             'room_id' => 'required',
-            'name' => 'required|min:1|max:50',
-            'name' => Rule::unique('webhooks')->ignore($this->id)->where(function ($query) {
-                return $query->where('user_id', \Auth::id());
-            }),
+            'name' => [ 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Requests/WebhookUpdateRequest.php</br>**Line**: 32</br>Whitespace found at end of line